hera_mc/geo_handling.py

```diff
@@ -329,7 +335,7 @@ def plot_stations(self, stations_to_plot, query_date, state_args):
             for a in self.session.query(geo_location.GeoLocation).filter(geo_location.GeoLocation.station_name == station):
                 show_it = True
                 if state_args['show_state'].lower() == 'active':
-                    show_it = self.is_in_connections(station, query_date)
+                    show_it = self.is_in_connections(station, query_date, False)
```

Comment:
I added the explicit return_antrev=False a al earlier.  I like keeping needed values in there, in case I foolishly change a default...